style.css

```diff
@@ -0,0 +1,78 @@
+/* nejsme vcerejsi, pouzivame moderni box model */
+
+html {
+    box-sizing: border-box;
+}
+
+*,
+ :after,
+ :before {
+    box-sizing: inherit;
+}
+
+body {
+    font-family: sans-serif;
+    font-size: 14px;
+}
+
+
+/* ---------------------------------------------------- */
+
+.zeny {
+    display: flex;
+    background-color: rgb(211, 211, 211);
+    border-radius: 30px;
+    max-width: 1300px;

```

Comment:
Tady maximální šířku nastavovat nepotřebujeme, nemáme to v zadání. Správně napsaný flexbox by se nám měl postarat o to, že se jeho položky roztáhnou přes celou šířku stránky 🙂